client/public/clientJS/agencyWishCards.js

```diff
@@ -0,0 +1,29 @@
+window.onload = function () {
```

Comment:
use $(document).ready from jquery for this

Reply:
Does this make any difference lol? (If following convention, I pretty much copied that from another js file, we aren't following any convention there :P).

I will change it though ;)